gen/roundtrip_test.go

```diff
@@ -69,8 +56,28 @@ func assertRoundTrip(t *testing.T, x thriftType, v wire.Value, msg string, args
 	}
 
 	gotX := reflect.New(xType).Interface().(thriftType)
-	if assert.NoError(t, gotX.FromWire(v), "FromWire: %v", message) {
-		return assert.Equal(t, x, gotX, "FromWire: %v", message)
+	if assert.NoError(t, gotX.FromWire(v), "FromWire: %encodedValue", message) {
+		return assert.Equal(t, x, gotX, "FromWire: %encodedValue", message)
 	}
+
 	return false
 }
+
+// assertBinaryRoundTrip checks that De/Encode returns the same value.
+func assertBinaryRoundTrip(t *testing.T, w wire.Value, message string) (wire.Value, bool) {
+	var buff bytes.Buffer
+	if !assert.NoError(t, protocol.Binary.Encode(w, &buff), "%encodedValue: failed to serialize", message) {
```

Comment:
typo: `%encodedValue` isn't a thing.

```suggestion
	if !assert.NoError(t, protocol.Binary.Encode(w, &buff), "%v: failed to serialize", message) {
```